SingularityService/src/main/java/com/hubspot/singularity/data/RequestManager.java

```diff
@@ -385,7 +390,31 @@ public SingularityDeleteResult deleteLbCleanupRequest(String requestId) {
   }
 
   public Optional<SingularityExpiringBounce> getExpiringBounce(String requestId) {
-    return getExpiringObject(SingularityExpiringBounce.class, requestId);
+    final Optional<SingularityExpiringBounce> optionalBounce = getExpiringObject(SingularityExpiringBounce.class, requestId);
```

Comment:
more of a convention thing, we tend to name vars for optionals as maybe..., so in this case `maybeExpiringBounce` or something similar